websockify/websocketproxy.py

```diff
@@ -297,12 +297,17 @@ def started(self):
         else:
             dst_string = "%s:%s" % (self.target_host, self.target_port)
 
+        if self.listen_fd != None:
+            src_string = "fd %d" % self.listen_fd
```

Comment:
This isn't terribly helpful for anyone. Perhaps just omit the entire "from ..." part if we don't know it?